pvoutput/asyncio/__init__.py

```diff
@@ -309,8 +313,10 @@ async def register_notification(self, appid: str, url: str, alerttype: int):
             raise TypeError(
                 f"alerttype needs to be an int, got: {str(type(alerttype))}"
             )
-        # TODO: urlencode the callback URL
 
-        call_url = f"https://pvoutput.org/service/r2/registernotification.jsp?appid={appid}&type={alerttype}&url={url}"
-        response = await (await self._call(endpoint=call_url, method="GET")).text()
-        return response
+        call_url, method = utils.URLS['registernotification']
```

Comment:
Needs to be `url, method` for consistency please.

Reply:
what do you want to call the function parameter url then? options:
1. rename input parameter to `callback_url`
2. introduce a new variable at the beginning of the function callback_url (if the function parameter cannot be changed)
`callback_url = url`

Reply:
🤦🏻 sorry I missed that, ignore this one 😇

Reply:
I think it might still be relevant, for consistency reasons as you mentioned, but can also be call_url in this case